### ISSUE TITLE
kde5: add printer

### ISSUE THREAD

**Issue by joelmo** [NixOS/nixpkgs#7588]
Using kde-add-printer installed from print-manager. I get this message when trying to add a printer:
```Failed to search for a recommended driver: 'The name org.fedoraproject.Config.Printing was not provided by any .service files'```

**Comment by ttuegel** [comment]
Is this the KDE 4 or KDE 5 print-manager?

**Comment by joelmo** [comment]
Ah it is kde 4, but I intended to install version 5.
```
[joelmo@jmdesk:~]$ nix-env -iA nixos.pkgs.kde5.print-manager
replacing old ‘print-manager-14.12.3’ installing ‘print-manager-14.12.3’ ... 
[joelmo@jmdesk:~]$ kde-add-printer -v Qt: 4.8.6 KDE Development Platform: 4.14.6 AddPrinter: 0.3.0
```

**Comment by ttuegel** [comment]
Nope, that's KDE 5. Many KDE 5 packages are still Qt 4-based.

**Comment by joelmo** [comment]
Shouldn't it say that the KDE platform is version 5?

**Comment by ttuegel** [comment]
No. Many "KDE 5" packages are also kdelibs-4-based.

**Comment by ttuegel** [comment]
This is fixed in KDE Applications 15.08.
